### PR TITLE
docs: improve Ruby documentation

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Fix white-space in ordered list
- Let the Markdown renderer deal with numbering the ordered list
- Use present tense
- Split up a sentence into two parts
- Remove "future work" section
- Reduce capitalization in headings and page title
- Use Gemfiles to denote multiples of `Gemfile`
- Remove outdated sentence talking about alpha support for Bundler
- Remove sentence that duplicates information already mentioned earlier in the document

## Context:

Helps with issue #7626.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] Unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/ruby.md [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
-title: Ruby Bundler Support
+title: Ruby Bundler support
 description: Ruby Bundler support in Renovate
 ---
 
-# Automated Dependency Updates for Ruby Bundler Dependencies
+# Automated dependency updates for Ruby Bundler dependencies
 
 Renovate supports upgrading dependencies in Bundler's `Gemfile`s and their accompanying `Gemfile.lock` files.
 Support is considered "alpha" stage until there have been some more real-world tests.
```
Comment (HonkingGoose) [inline docs/usage/ruby.md:9]:
```suggestion

```

Can we maybe remove this sentence? What is the status of Ruby support for Renovate right now?

File: docs/usage/ruby.md [hunk 2]
```diff
@@ -1,29 +1,23 @@
...
-
-## Future work
-
-- Updating `.gemspec` files
-- Pinning dependencies to the version found in `Gemfile.lock` rather than the latest matching version
-- Lock file maintenance
-- Selective lock file updating (if ranges are in use in the `Gemfile`)
+You can install the [Renovate App](https://github.com/apps/renovate) on GitHub.
+Or you can check out [Renovate OSS](https://github.com/renovatebot/renovate) to self-host Renovate.
+Both versions support Bundler.
```
Comment (HonkingGoose) [inline docs/usage/ruby.md:23]:
```suggestion

```

Maybe we can scrap this sentence?

The top section of this document mentions Ruby bundler support already in the `h1` heading: `# Automated dependency updates for Ruby Bundler dependencies`

File: docs/usage/ruby.md (hunk 1 above)
Comment (JamieMagee) [reply docs/usage/ruby.md:9]: Probably. We have a `releaseStatus` property which can be `alpha` or `beta`. Bundler is neither of those, and hasn't seen above average number of issues or bug fixes IMO.

File: docs/usage/ruby.md [hunk 3]
```diff
@@ -1,29 +1,23 @@
...
-## How It Works
+## How it works
 
-1.  Renovate will search each repository for any `Gemfile` files
-2.  Existing dependencies will be extracted from the files
-3.  Renovate will resolve the dependency on Rubygems or elsewhere if configured, and look for any newer versions
-4.  A PR will be created with `Gemfile` and `Gemfile.lock` updated in the same commit
-5.  If the source repository has either a "changelog" file or uses GitHub releases, then Release Notes for each version will be embedded in the generated PR
+1. Renovate searches in each repository for any `Gemfile` files
+1. Existing dependencies are extracted from the `Gemfile`'s
```
Comment (JamieMagee) [inline docs/usage/ruby.md:14]: Plural, not possessive?

```suggestion
1. Existing dependencies are extracted from the `Gemfile`s
```

Comment (JamieMagee) [reply docs/usage/ruby.md:14]: Also, is it "Gemfile files" or "Gemfiles"? Honestly, I'm not sure.

File: docs/usage/ruby.md (hunk 1 above)
Comment (rarkins) [reply docs/usage/ruby.md:9]: Yes, let's remove it.l

File: docs/usage/ruby.md (hunk 3 above)
Comment (HonkingGoose) [reply docs/usage/ruby.md:13]: Let's use Gemfiles, that's what https://bundler.io/ uses on their own webpage. 😄

**Review by JamieMagee — APPROVED** [review]

**Pull request merged**